mailpoet/views/welcome_wizard.html

```diff
@@ -4,7 +4,6 @@
 <script>
   var mailpoet_logo_url = '<%= cdn_url('welcome-wizard/mailpoet-logo.20200623.png') %>';
   var wizard_sender_illustration_url = '<%= cdn_url('welcome-wizard/sender.20200623.png') %>';
-  var wizard_email_course_illustration_url = '<%= cdn_url('welcome-wizard/email-course.20200623.png') %>';
```

Comment:
Was wondering if it makes sense to keep the image(s) in the repo, I guess we can remove them as well, what do you think?

Reply:
Those images are not a part of the build, so I think we don't delete them. If I'm not wrong, they were uploaded to wordpress.org. You can see quite old images in [the folder](https://github.com/mailpoet/mailpoet/tree/trunk/mailpoet/plugin_repository/assets/welcome-wizard), so I guess that it's fine.